control/pid.go

```diff
@@ -126,40 +210,93 @@ func (p *basicPID) reset() error {
 	}
 
 	p.tuning = false
-	if p.kI == 0.0 && p.kD == 0.0 && p.kP == 0.0 {
-		var ssrVal float64
-		if p.cfg.Attribute["tune_ssr_value"] != nil {
-			ssrVal = p.cfg.Attribute["tune_ssr_value"].(float64)
-		}
+	if p.useMulti {
+		for i := 0; i < len(p.PIDSets); i++ {
 
-		tuneStepPct := 0.35
-		if p.cfg.Attribute.Has("tune_step_pct") {
-			tuneStepPct = p.cfg.Attribute["tune_step_pct"].(float64)
-		}
+			if p.PIDSets[i].I == 0.0 && p.PIDSets[i].D == 0.0 && p.PIDSets[i].P == 0.0 {
```

Comment:
we have the [NeedsAutoTuning() ](https://github.com/viamrobotics/rdk/blob/ea22dd9fc5c69029c1b4e3bff1058a4c6b9adef9/control/setup_control.go#L55-L58)function on PIDConfig that you can use here